phasespace/fulldecay/__init__.py

```diff
@@ -0,0 +1,15 @@
+import sys
+
+from .fulldecay import FullDecay
+
+try:
+    import zfit
+    import zfit_physics as zphys
+    from particle import Particle
+except ModuleNotFoundError:
+    print(
+        "the fulldecay functionality in phasespace requires particle and zfit-physics. "
+        "Either install phasespace[fulldecay] or particle and zfit-physics.",
+        file=sys.stderr,
+    )
+    raise

```

Comment:
```suggestion
except ModuleNotFoundError as error:
    raise ModuleNotFoundError(
        "The fulldecay functionality in phasespace requires particle and zfit-physics. "
        "Either install phasespace[fulldecay] or particle and zfit-physics.",
        file=sys.stderr,
    ) from error

```

I would write it like that. Otherwise, the print is simply overlooked